src/App.jsx

```diff
@@ -23,6 +24,9 @@ function App() {
           <Route path="/app" element={<AppLayout />}>
             <Route index element={<Home />} />
           </Route>
+          <Route path="/history" element={<AppLayout />}>  
+            <Route index element={<ClinicalHistory />} />
+          </Route>
```

Comment:
This should go inside the /app path to avoid re-rendering. The new URI would be /app/history

          <Route path="/app" element={<AppLayout />}>
            <Route index element={<Home />} />
            <Route path="/history" element={<ClinicalHistory />} />
          </Route>